chia/plotting/util.py

```diff
@@ -229,8 +230,19 @@ def get_filenames(directory: Path, recursive: bool) -> list[Path]:
         return []
     all_files: list[Path] = []
     try:
-        glob_function = directory.rglob if recursive else directory.glob
-        all_files = [child for child in glob_function("*.plot") if child.is_file() and not child.name.startswith("._")]
+        if follow_links and recursive:
+            import glob
+
+            files = glob.glob(str(directory / "**" / "*.plot"), recursive=True)
+            for file in files:
+                filepath = Path(file).resolve()
+                if filepath.is_file() and not filepath.name.startswith("._"):
```

Comment:
Is this looking for files starting with `._`?  Or files starting with either `.` or `_`.

```python-console
>>> "_.".startswith("._")
False
```

i see this check already exists as well, so this may be catching some exisitng bug.  mildly tempted to encourage deduping that.  maybe.  maybe not.

Reply:
Yea I left it as is - I assumed it was looking specifically or "._" for some reason that is unknown